backend/api/publishing.py

```
@@ -394,6 +690,87 @@ async def save_scenario_draft(
         db.rollback()
         raise HTTPException(status_code=500, detail=f"Failed to save scenario: {str(e)}")
 
+@router.put("/{scenario_id}/status")
+async def update_scenario_status(
+    scenario_id: int,
+    status_data: dict,
+    db: Session = Depends(get_db),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)